waterlog/src/Components/SegmentLeak.js

```diff
@@ -72,13 +76,13 @@ class SegmentLeak extends Component {
 		
 		const leakInfo = (
 			<div>
-				<div className={`leakInfo ${parseInt(alert.status) == 2 ? alert.severity : 'leak-resolved'}`}>
-					<h2>{`${alert.entityName} ${alert.entityId} ${alert.entityType}`}</h2>
-					<p>({alert.severity})</p>
-					<h1>R {alert.cost.toFixed(2)}</h1> 
-					<p>is being lost per hour!</p>
-					<p>Loosing {alert.litresPerHour.toFixed(0)}&#x2113; per hour</p>
-					<p>no leak would be 0&#x2113; per hour</p>
+				<div>
+					<h2 className={this.getSeverityClass(alert.severity)}>{`${alert.entityName} ${alert.entityId} ${alert.entityType}`}</h2>
```

Comment:
call the function once, assign its value to a variable and assign that variable to the class

Reply:
Done please re-check